site/using-weave/service-management.md

```diff
@@ -38,6 +39,17 @@ With the above in place, you are ready to connect to the 'nc' service from anywh
 
 Similar NAT rules to the above can used to expose services not just to the outside world but also other, internal, networks.
 
+###<a name="export-route"></a> Exporting Services to the wider network
+
+If you want hosts that are not running Weave Net to access services
+exposed as above, on each of those machines:
+
+    ip route add <network-cidr> via <exposing-host>
+
+Where `<network-cidr>` gives the IP address range used by weave,
+e.g. `10.2.0.0/16` or `10.32.0.0/12` and `<exposing-host>` is the
+address of the machine that has run `weave expose`, e.g. `host2` in
+the example above.
```

Comment:
Where, 

* `<network-cidr> is the IP address range that is used by Weave, for example, 10.2.0.0/16` or `10.32.0.0/12` and, 
* `<exposing-host>` is the address of the machine on which `weave expose` was run. For example, `host2` in the example above <-- not sure which example this is referring to?